migrator/postgreshelper/utils.go

```diff
@@ -0,0 +1,45 @@
+package postgreshelper
```

Comment:
Moved this from pgAdmin to the migrator to isolate it to where it is used to more easily remove it later.